docs/changes.rst

```diff
@@ -57,6 +57,14 @@ Security fixes:
 
 - fix security issue with remote repository access, #1428
 
+Bug fixes:
+
+- do not write objects to repository that are bigger than the allowed size,
+  borg will reject reading them, #1451.
+  IMPORTANT: if you created archives with many millions of files or
+             directories, please verify if they extract successfully (e.g. try
+             a "borg extract --dry-run").
```

Comment:
A `borg list repo::archive in question` or check --archives-only would be sufficient.